devops/mac.yml

```diff
@@ -0,0 +1,66 @@
+trigger: none
```

Comment:
This is the 3OS file, maybe rename to something other than mac.yml?

Reply:
I'll delete 3os and the mac.yml .